marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers.hs

```diff
@@ -78,7 +86,9 @@ buildIndexers catchupConfig utxoConfig mintEventConfig logger path = do
   mainCoordinator <-
     lift $ standardCoordinator mainLogger [blockInfoWorker, coordinatorTxBodyWorkers]
 
-  pure (queryIndexer, mainCoordinator)
+  resumePoints <- Core.lastSyncPoints (fromIntegral securityParam + 1) mainCoordinator
+
+  pure (resumePoints, queryIndexer, mainCoordinator)
```

Comment:
Add a todo to create a dedicated return type for it. We probably want to wait until we have more stuff in the query side before we do it (unless you want to create this type now and we'll update it).

Reply:
I think it makes more sense to do it once, like you said, we have more stuff on the query side.